extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/AssetEntryDto.java

```diff
@@ -2,23 +2,23 @@
 
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
+import java.util.Map;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-import java.util.Map;
-
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder(toBuilder = true)
 public class AssetEntryDto {
     @Schema(requiredMode = RequiredMode.REQUIRED)
-    private String assetRequestId;
-    private Map<String, Object> assetRequestProperties;
+    private String id;
+    private Map<String, Object> assetProperties;
+    private Map<String, Object> privateAssetProperties;
```

Comment:
Map<String, Object> is a polymorphism, either we need good evidence how this works in the API client libraries, or we should skip this in favor of something that will make the user handle it themselves, but doesn't interfer with the api client generation, e.g. String assetPropertiesJson